### ISSUE TITLE
Error starting tasks

### ISSUE THREAD

**Issue by potsed** [chadrien/sublime-toggl#4]
This has just started happening recently, not sure if it's an API change or something similar, here is a log:

```python
Traceback (most recent call last):
  File "toggl.commands.manage_time_entries_window in C:\LocalApps\Sublime Text\Data\Installed Packages\Toggl Timer.sublime-package", line 53, in <lambda>
  File "toggl.commands.manage_time_entries_window in C:\LocalApps\Sublime Text\Data\Installed Packages\Toggl Timer.sublime-package", line 56, in start_time_entry
  File "toggl.utils.toggl_api.toggl_time_entry_api in C:\LocalApps\Sublime Text\Data\Installed Packages\Toggl Timer.sublime-package", line 8, in start
  File "toggl.utils.toggl_api.toggl_base_api in C:\LocalApps\Sublime Text\Data\Installed Packages\Toggl Timer.sublime-package", line 40, in request
  File "./urllib/request.py", line 156, in urlopen
  File "./urllib/request.py", line 475, in open
  File "./urllib/request.py", line 587, in http_response
  File "./urllib/request.py", line 513, in error
  File "./urllib/request.py", line 447, in _call_chain
  File "./urllib/request.py", line 595, in http_error_default
urllib.error.HTTPError: HTTP Error 400: Bad Request
```

**Comment by phaberest** [comment]
Same here

**Comment by franckdelage** [comment]
Same here, using ST3 build 3086.

**Comment by silverbackis** [comment]
Same here still.

**Comment by silverbackis** [comment]
Found the issue. It is the encoding of JSON in the post using the latest version of Python bundled with Sublime Text 3.

Install 'PackageResourceViewer' and edit the following:
/toggl/utils/toggle_api/toggle_base_api.py

Change line 55 which returns data in the 'prepare_datas' function to:
return json.dumps(datas).encode('utf-8')

In: /toggl/utils/toggle_api/toggle_base_api.py
Immediately after 'application/json' in the Content-Type add '; charset=utf-8' - do this on lines 8 and 11. parameter for the headers should look like this:
{'Content-Type': 'application/json; charset=utf-8'}

As a bonus add the following to the start data call:
"created_with":"Sublime Text 3" so you know an entry is created via the package.

Line 8 in it's entireity:
return self.get_response(self.request(self.TIME_ENTRY_START_URL, self.prepare_datas({'time_entry': {'description': description, 'pid': project_id, "created_with":"Sublime Text 3"}}), {'Content-Type': 'application/json; charset=utf-8'}, 'POST'))['data']

It's a hack to change it by extracting the packed packaged and overwriting the file so hopefully it'll be implemented when this is spotted.

**Comment by sequielo** [comment]
Based on @silverbackis hacks, I implemented the pull request mentioned above #7

**Comment by chadrien** [comment]
Closed by #7
